stix2/datastore/__init__.py

```diff
@@ -129,7 +149,7 @@ def relationships(self, *args, **kwargs):
             obj (STIX object OR dict OR str): The STIX object (or its ID) whose
                 relationships will be looked up.
             relationship_type (str): Only retrieve Relationships of this type.
-                If None, all relationships will be returned, regardless of type.
+            If None, all relationships will be returned, regardless of type.
```

Comment:
trivial issue, but I think this should remain indented.

Reply:
Oversight, I prefer the indent as well.